actors/multisig/src/lib.rs

```diff
@@ -472,7 +472,11 @@ fn execute_transaction_if_approved(
             Ok(Some(r)) => {
                 out = RawBytes::new(r.data);
             }
-            Err(e) => {
+            Err(mut e) => {
+                if let Some(r) = e.take_data() {
+                    out = RawBytes::new(r.data);
+                }
+
```

Comment:
@arajasek One question:

Why do we allow the transaction to "succeed" i.e. return an Ok value from this function even though the transaction fails ?

Reply:
Expanding just a bit, I think "philosophically" it makes sense to succeed here. When this method is called (or a top-level "approve and execute" method is called), you're asked to _execute_ the transaction, if possible. If for some reason the transaction isn't executable, _that's_ a failure. 

But if the transaction executes-and-fails, that's not really a failure for the top-level viewer. We were asked to execute the tx, we were able to do so, and the _result_ of that execution happened to be a failure.

But @Stebalien is right, there are definitely users who won't want this behaviour, so getting this out of built-in actors would be great.